modules/rosapps/applications/screensavers/blankscr/scrnsave.rc

```diff
@@ -58,4 +61,4 @@ IDI_ICON  ICON  DISCARDABLE  "res/icon_blankscr.ico"
 #endif
 #ifdef LANGUAGE_ZH_CN
     #include "lang/zh-CN.rc"
-#endif
+#endif
```

Comment:
By the way, a blank line after the end of the file is required as per the translation guidelines [here](http://reactos.org/wiki/Translation_Introduction#How_to_translate_the_files).